graph-commons/src/test/scala/io/renku/http/client/GitLabClientSpec.scala

```diff
@@ -51,99 +50,131 @@ class GitLabClientSpec
     with should.Matchers
     with MockFactory {
 
-  "send" should {
+  "get" should {
 
     "fetch json from a given page" in new TestCase {
 
       stubFor {
-        callMethod(method, s"/api/v4/$path")
+        callMethod(GET, s"/api/v4/$path")
           .willReturn(
             okJson(result)
               .withHeader("X-Next-Page", maybeNextPage.map(_.show).getOrElse(""))
           )
       }
 
-      client.send(method, path, endpointName)(mapResponse).unsafeRunSync().toString() shouldBe result
+      client.get(path, endpointName)(mapGetResponse).unsafeRunSync().toString() shouldBe result
     }
 
     "fetch commits using the given access token" in new TestCase {
```

Comment:
I've just noticed that the test desc somehow necessarily mentions `commits` while on the GitLabClient level this sort of concreteness is probably not what we want.